submit_compress.csh

```diff
@@ -38,26 +38,28 @@
 # #PBS  -l select=18:ncpus=36:mpiprocs=36
 #PBS  -l walltime=00:20:00
 #PBS  -o compress.out
+#PBS  -k eod
 #PBS  -j oe 
 
+# Submit from CASEROOT instead of the rest directory,
+# so that the job output file doesn't clutter the data directory
+# and data_scripts.csh is easier to find.
+
```

Comment:
Can we check if the data_scripts.csh file exists, or does it automatically die if the `source data_scripts.csh` fails?